requirements.txt

```diff
@@ -39,7 +43,7 @@ pyparsing==2.4.7
 pytest==6.1.2
 pytest-cov==2.10.1
 pytest-django==4.1.0
-python-dateutil==2.8.1
+python-dateutil==2.6.0
```

Comment:
What caused this downward bump and should it be worth of mentioning here?

Reply:
There was a version conflict for the dateutil dependency required by djago auditlog and what was used previously. This is how pip resolved the conflict and everything seems to work. I didn't find any real problems with just forcing this to 2.8.1 but it produces a nasty looking warning and would probably need more thorough testing.

Reply:
Alrighty :+1:  It would be nice to have a comment in the requirements that we're sticking to this version by a reason.